unlock-app/src/components/interface/checkout/CryptoCheckout.tsx

```diff
@@ -213,6 +221,10 @@ export const CryptoCheckout = ({
       !hasValidkey) ||
     (isAdvanced && hasValidKeyOrPendingTx && !transactionPending)
 
+  const onLoading = useCallback((loading: boolean) => {
```

Comment:
do we need useCallback? There are no dependencies. It can simply be a named function.